Source/WebCore/css/html.css

```diff
@@ -892,7 +873,6 @@ input[type="range"]::-webkit-slider-runnable-track {
 }
 
 input[type="range"]::-webkit-slider-thumb, input[type="range"]::-webkit-media-slider-thumb {
-    appearance: auto;
```

Comment:
Sorry, I missed that these are also `<div>` (should have caught this earlier)

Reply:
(e.g. this one is still needed)

Reply:
I should have checked on this after your last comment on <div>. Sorry about this. Patch has been updated.